### DISCUSSION TITLE
This installer is deprecated.

### DISCUSSION THREAD

**Discussion by m1n9o** [python-poetry/poetry#4820] [Q&A]
I am trying to follow official document to start poetry project on my mac.
When I install poetry.
```
curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
```
It says.
```
Retrieving Poetry metadata

This installer is deprecated. Poetry versions installed using this script will not be able to use 'self update' command to upgrade to 1.2.0a1 or later.
```
So what's the latest installer? Can we use brew to install?

**Comment by abn** [comment]
The `get-poetry` script is being deprecated in favour of the [`install-poetry.py`](https://github.com/python-poetry/install.python-poetry.org/blob/main/install-poetry.py) script as described [here](https://python-poetry.org/docs/#installation).

The short answer is you can now use the following.

```sh
curl -sSL https://install.python-poetry.org/ | python -
```

**Reply by uSpike** [reply]
I'm not in favor of instructing users to only use an un-versioned `curl | python` script from a website.  This is a major security issue if this server were to start responding with malicious code.  For CI and security purposes, myself and the organizations I'm part of will continue using the `https://raw.githubusercontent.com` URL with a specific tag/commit.  Please support this use case indefinitely.

**Reply by KotlinIsland** [reply]
@abn

This should be a big red warning message indicating that it's deprecated, not an innocent looking info bubble.
![image](https://user-images.githubusercontent.com/65446343/148184204-7e343b4a-0fd1-42c1-8804-bd693918e7cc.png)
